app/views/companies/index.html.haml

```diff
@@ -14,37 +14,40 @@
       #{t('.h_companies_listed_below')}
       #{t('.how_to_search')}
 
-    .panel.panel-default{ style: 'padding-left: 5px; padding-right: 5px;' }
-      = render 'search_form'
+  .panel.panel-default{ style: 'padding-left: 5px; padding-right: 5px;' }
+    = render 'search_form'
 
-  %table.table.table-hover
-    %thead
-      %tr
-        %th
-          = sort_link(@search_params, :business_categories_name, t('.category'))
-        %th
-          = sort_link(@search_params, :name, t('.name'))
-        %th
-          = sort_link(@search_params, :region_name, t('.region_land'))
-        %th
-          = sort_link(@search_params, :city,
-                      t('activerecord.attributes.company.city'))
-        - if current_user.try(:admin)
+  - if @companies.empty?
+    %strong No records matched your search criteria.
```

Comment:
This needs to be t() - right? 

"Din sökning gav inget resultat, prova ta bort något av dina val"